### DISCUSSION TITLE
How to MinIO single cluster Node add

### DISCUSSION THREAD

**Discussion by leejamein** [minio/minio#19266] [Q&A] [answered]
I am using a MinIO single POOL. How do I add a Node to the POOL I am using?

Existing settings in use
`MINIO_VOLUMES="https://minio-0{1...4}.test.com:9000/mnt/disk{1...4}/minio"`

Changed settings
`MINIO_VOLUMES="https://minio-0{1...5}.test.com:9000/mnt/disk{1...4}/minio"`

result

  Loaded: loaded (/usr/lib/systemd/system/minio.service; enabled; vendor preset: disabled)
   Active: failed (Result: exit-code) since Wed 2024-03-13 17:13:25 KST; 1 day 16h ago
     Docs: https://min.io/docs/minio/linux/index.html
  Process: 1089124 ExecStart=/usr/local/bin/minio server $MINIO_OPTS $MINIO_VOLUMES (code=exited, status=1/FAIL>
  Process: 1089122 ExecStartPre=/bin/bash -c if [ -z "${MINIO_VOLUMES}" ]; then echo "Variable MINIO_VOLUMES no>
 Main PID: 1089124 (code=exited, status=1/FAILURE)

Is there a way to add a node to an existing POOL?

The newly added Node has the same configuration as the existing Nodes.

**Comment by harshavardhana** [comment, marked as answer]
No it is not possible.
